### DISCUSSION TITLE
Option to change the CTRL + ENTER shortcut to something user defined?

### DISCUSSION THREAD

**Discussion by mfobert** [community/community#6727] [Copilot Conversations] [answered]
Hi there,

I'm wondering if it is possible for a user to change the CTRL + ENTER shortcut in VSCode to something else? I have an extension I wrote ([StackFinder](https://marketplace.visualstudio.com/items?itemName=mark-fobert.stackfinder)) that uses the same key-binding and if you aren't going to add this option I will have to consider changing the key-binding on mine, as I think you're going to be a little more popular haha!

Thanks,
Mark

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Please check our documentation - https://github.com/github/copilot-docs/blob/main/docs/configuringvisualstudiocode.md#keyboard-shortcuts

**Reply by jeffreyyjp** [reply]
The link is 404

**Reply by dannyskoog** [reply]
https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#8-keyboard-shortcuts
